django/thunderstore/repository/models/package_installer.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+
+class PackageInstaller(models.Model):
+    identifier = models.SlugField(unique=True, db_index=True, editable=False)
```

Comment:
`db_index` is a bit redundant here since `unique` already guarantees the creation of index.